src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -90,33 +63,11 @@ void executeNextCommandTarget()
             switch ((VSConstants.AppCommandCmdID)commandId)
             {
                 case VSConstants.AppCommandCmdID.BrowserBackward:
-                    ExecuteBrowserBackward(subjectBuffer, contentType, executeNextCommandTarget);
+                    ExecuteBrowserBackward(executeNextCommandTarget);
```

Comment:
Should we add editor commanding support for these commands so you could switch to modern command handlers and get rid of this command filter altogether?

Reply:
it would be great to start a discussion on that so we can remove this entirely.